.github/ISSUE_TEMPLATE/funding_application.yml

```diff
@@ -0,0 +1,220 @@
+    id: tools
+    attributes:
+      label: Are there tools or tech that still need to be produced to facilitate the
+        funding initiative?
+      placeholder: null
+    validations:
+      required: false
+  - type: textarea
+    id: requirements
+    attributes:
+      label: Give a summary of the requirements that contextualize the costs of the
+        funding initiative
+    validations:
+      required: true
+  - type: markdown
+    attributes:
+      value: Covering the planning phase, this next section should provide 
+        a relatively detailed plan of how work will start on the funding initiative. This
+        section is important because funding initiatives need to get up and running
+        quickly.
+  - type: markdown
+    attributes:
+      value: Funding Initiative Plan - It is important to think about setting up collaboration
+        platform for code for contributors, reporting framework, license
+        decisions, etc.
+  - type: input
+    id: responsible
+    attributes:
+      label: Who is responsible for doing the work of this funding initiative?
+      description: This person is the primary contact to oversee the idea.
+      placeholder: null
+    validations:
+      required: true
+  - type: input
+    id: accountable
+    attributes:
+      label: Who is accountable for doing the work of this funding initiative?
+      placeholder: null
+    validations:
+      required: true
+  - type: input
+    id: backup
+    attributes:
+      label: If the responsible or accountable parties are no longer available, what
+        is the backup contact or plan?
+      placeholder: null
+    validations:
+      required: true
+  - type: input
+    id: WG
+    attributes:
+      label: Which technical initiative will this funding initiative be associated
+        with, and will it report to which WG or project?
+      placeholder: null
+    validations:
+      required: true
+  - type: input
+    id: license
+    attributes:
+      label: What license is this funding initiative being used under?
+      placeholder: null
+    validations:
+      required: true
+  - type: markdown
+    attributes:
+      value: The final section is to list the Milestones and Approximate costs
+        associated with the funding initiative. If you prefer, you can submit a
+        Google Sheet or Google Doc in the entry below.
+  - type: textarea
+    id: Milestones
+    attributes:
+      label: List the major milestones and timelines the funding initiative plans to
+        have and the associated payments with each.
+      placeholder: By the end of quarter 2 of 2025, the GA release will have 500 users
+        surveyed for performance assessment completing Milestone 1 for a release
+        of 5k in funding.
+    validations:
+      required: true
+  - type: checkboxes
+    id: terms
+    attributes:
+      label: Code of Conduct
+      description: By submitting this issue, you agree to follow our [OpenSSF Code of
```

Comment:
It feels to me like this should be up in the previous section and specifically ensuring that the TI itself is following the OpenSSF code of conduct "…to ensure an inclusive environment for participants."

Reply:
Moved under licenses.